app/src/components/App.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+function App(){
+    return (
+        <div className="container">
+            <h1>It's cat!!!!</h1>
+
+            <hr/>
```

Comment:
do not use `hr` tag plz. if u need border - use css border

Reply:
Done